packages/expo-asset/src/AssetSourceResolver.web.ts

```diff
@@ -84,7 +84,7 @@ export default class AssetSourceResolver {
     return this.fromSource(getScaledAssetPath(this.asset));
   }
   scaledAssetURLNearBundle(): ResolvedAssetSource {
-    const path = this.jsbundleUrl || 'file://';
```

Comment:
Prepending `file://` to the web asset path actually breaks it. This is because `getScaledAssetPath` returns the `httpServerLocation` which already has a protocol prefix